reproserver/web/templates/base.html

```diff
@@ -1,66 +1,67 @@
-              <li class="nav-item"><a class="nav-link{% if current_nav == 'webcapture' %} active{% endif %}"{% if current_nav == 'webcapture' %} aria-current="true"{% endif %} href="{{ reverse_url('webcapture_index') }}">Web capture</a></li>
-              <li class="nav-item ms-md-auto"><a class="nav-link{% if current_nav == 'about' %} active{% endif %}"{% if current_nav == 'about' %} aria-current="page"{% endif %} href="{{ reverse_url('about') }}">About</a></li>
-              <!--dropdown-->
-              <li class="nav-item dropdown">
-                <a href="#" class="nav-link dropdown-toggle" role="button" data-bs-toggle="dropdown" aria-expanded="false">Links<span class="caret"></span></a>
-                <ul class="dropdown-menu dropdown-menu-md-end">
-                  <li><a class="dropdown-item" href="https://github.com/VIDA-NYU/reproserver">GitHub Repository</a></li>
-                  <li><a class="dropdown-item" href="https://www.reprozip.org/">ReproZip Website</a></li>
-                </ul>
-              </li>
-              <!--end dropdown-->
-            </ul>
+<body>
 
-          </div><!-- /.navbar-collapse -->
-        </div><!-- /.container -->
-      </nav><!-- /.navbar -->
+  <div class="container-fluid" id="content" role="main">
+    <div class="row">
+    <!-- Menubar -->
+    <nav class="navbar navbar-expand-md navbar-dark bg-primary">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="/">ReproServer</a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarContent"
+          aria-controls="navbarContent" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarContent">
+          <ul class="navbar-nav mb-2 ms-auto">
+            <li class="nav-item"><a class="nav-link{% if current_nav == 'about' %} active{% endif %}" {% if
+                current_nav=='about' %} aria-current="page" {% endif %} href="{{ reverse_url('about') }}">About</a></li>
+            <!--dropdown-->
+            <li class="nav-item dropdown">
+              <a href="#" class="nav-link dropdown-toggle" role="button" data-bs-toggle="dropdown"
+                aria-expanded="false">Links<span class="caret"></span></a>
+              <ul class="dropdown-menu dropdown-menu-md-end">
+                <li><a class="dropdown-item" href="https://github.com/VIDA-NYU/reproserver">GitHub Repository</a></li>
+                <li><a class="dropdown-item" href="https://www.reprozip.org/">ReproZip Website</a></li>
+              </ul>
+            </li>
+            <!--end dropdown-->
+          </ul>
+        </div><!-- /.navbar-collapse -->
+      </div><!-- /.container -->
+    </nav><!-- /.navbar -->
 
-      <div class="container mt-3"> <!--Body content-->
```

Comment:
This removes the margin between the top navbar and the breadcrumbs

![screenshot](https://user-images.githubusercontent.com/426784/203159579-71e59e5e-2ec8-45e9-8972-ec39fdc37495.png)

Reply:
fixed in latest commit - 9eccd033e13934b57d254afdce74106739d07f1e